.github/CODEOWNERS

```
@@ -0,0 +1,2 @@
+# Global owners
+*       @naruse666 @R1013-T @waya279
```

```suggestion
*       @Peace Net
```

これチーム名にスペース入ってるけどいけるのか？

---

`peace-net`こっちですかね？

---

`https://github.com/orgs/TECH-C-LT/teams/peace-net`

---

`@TECH-C-LT/peace-net`こうですかね

---

サジェスト変更してますmm